roles/config-rh-sso/tasks/realm.yml

```diff
@@ -18,27 +18,16 @@
   set_fact:
     realm: "{{ lookup('template', 'realm.json.j2') | to_json }}"
 
-- name: "Validate existing Realms"
-  uri:
-    url: "http://{{ external_ip }}:8080/auth/admin/realms/{{ item.name }}"
-    method: GET
-    headers:
-      Authorization: "Bearer {{ rh_sso_token.json.access_token }}"
-  register: existing_realms
-  ignore_errors: true
-
 - name: "Create Realm for service Red Hat Single Sign-On"
   uri:
     url: "http://{{ external_ip }}:8080/auth/admin/realms"
     method: POST
     body: "{{ realm }}"
     body_format: json
-    status_code:
-     - 201
+    status_code: 201, 422, 409
```

Comment:
@BinaryDevotee Can you explain a bit about what status `422` and `409` are? I'm assuming something like "This realm already exists".. just want to think a bit about what else may trigger them (i.e. the realm exists, but the settings don't necessarily match, etc.)

Reply:
@tylerauerbeck I only need 201 and 409 for this to work. The 409 error code means conflict, when the entity already exists on the remote server, which would point to an already existing Realm.
By adding the 409 error code on the URI module, it creates if it doesn't exist.
I removed the error code 422, which is no longer necessary for this.